src/steps/02-EvaluationCriteria/MRR/CertificationPOCTypeForm.vue

```diff
@@ -0,0 +1,384 @@
+<template>
+  <div>
+    <ATATRadioGroup
+        v-if="loaded"
+        :id="POCType + 'certificationPOCType'"
+        :legend="sequence + '. Tell us about your ' + POCType + ' POC'"
+        legend-as-subheader="true"
+        :helpText="'Will any of these individuals serve as the '
+              + POCType + ' Certifier for your J&A?'"
+        :value.sync="certificationPOCType"
+        :items="certificationPOCTypeOptions"
+        :name="'certification-poc-' + POCType + '-radio-group'"
+        class="copy-max-width mb-10 mt-3"
+        :rules="[$validators.required('Please select an option')]"
+    />
+    <ATATContactForm
+        v-if="showContactForm"
+        :email.sync="email"
+        :firstName.sync="firstName"
+        :lastName.sync="lastName"
+        :middleName.sync="middleName"
+        :phone.sync="phone"
+        :phoneExt.sync="phoneExt"
+        :selectedBranch.sync="selectedBranch"
+        :selectedPhoneCountry.sync="selectedPhoneCountry"
+        :selectedRank.sync="selectedRank"
+        :selectedRole.sync="selectedRole"
+        :selectedSalutation.sync="selectedSalutation"
+        :suffix.sync="suffix"
+        :loaded="loaded"
+        :roles.sync="contactRoles"
```

Comment:
The Figma includes the field "Job Title" but does not include "Email address" for Civilian. For Military, the Figma still includes "Job Title" which I'm 99% sure is only bc the same component was used for the prototype. Will verify with Melissa or JC exactly which fields to include in both cases. 
![Screen Shot 2023-05-18 at 2 52 56 PM](https://github.com/dod-ccpo/atat-web-ui/assets/90333349/4b2127df-0e17-4418-ae3c-84292cfdf7b6)